src/Oro/Bundle/EmbeddedFormBundle/Resources/public/js/embed.form.js

```diff
@@ -77,10 +77,11 @@ var ORO = (function(ORO) {
                     }
 
                     realTag = document.createElement('script');
+                    for (var i = 0; i < scripts[k].attributes.length; i++) {
```

Comment:
Element `<script src="">` may not have `async` attribute, so `scripts[k].hasAttribute('async')` return false. Without  `async` attribute the script tag load synchronously, this ensures that scripts are executed in correct order, but `document.createElement('script')` loads the JS asynchronously by default. To it fix needs also add a new checks

```js
if (!scripts[k].hasAttribute('async')) {
    realTag.async = false;
}
```

Reply:
Looks like you are right. Have checked the issue here https://jsfiddle.net/hoybuLej/42/

Let's add code that you have proposed.